src/voodoo-prep/dune

```diff
@@ -2,4 +2,4 @@
  (name prep)
  (public_name voodoo-prep)
  (package voodoo-prep)
- (libraries cmdliner fpath bos opam-format))
+ (libraries cmdliner fpath bos opam-format voodoo-lib))
```

Comment:
This would have the consequence that any change to `voodoo-lib` causes the pipeline to re-run all the voodoo-prep jobs - which is costly. So we better not make `voodoo-prep` depend on `voodoo-lib`.

Reply:
Done, I only reworked the Package.t type